packs/bestiary-effects/effect-lawbringer.json

```diff
@@ -0,0 +1,57 @@
+{
+    "_id": "bRYIsyDIsZNaLUEe",
+    "img": "icons/commodities/treasure/token-gold-gem-purple.webp",
+    "name": "Effect: Lawbringer",
+    "system": {
+        "description": {
+            "value": "<p>The shieldmarshal marks a foe for judgment.</p>"
+        },
+        "duration": {
+            "expiry": null,
+            "sustained": false,
+            "unit": "unlimited",
+            "value": -1
+        },
+        "level": {
+            "value": 1
+        },
+        "publication": {
+            "license": "OGL",
+            "remaster": false,
+            "title": "Pathfinder #180: The Smoking Gun"
+        },
+        "rules": [
+            {
+                "key": "TokenMark",
+                "slug": "lawbringer"
+            },
+            {
+                "domain": "all",
+                "key": "RollOption",
+                "label": "PF2E.NPCAbility.Shieldmarshal.LawbringerSpeedLabel",
```

Comment:
I considered having the speed bonus on the Lawbringer abilities, particularly for the Gunmarshal as it doesn't receive the damage bonus and thus doesn't really need the TokenMark.

However, I figured it might be useful even for them if in the future we gain the ability to see the TokenMarks for the currently selected actor.